### PR TITLE
Add special comments used by starfall for the generation of documentation.

### PR DESCRIPTION
**Pull request by Syvil**
This should provide the information needed by Starfall to generate documentation pages for the library and functions in the PvP section of the CFC Starfall library. The documentation should appear when viewing the Starfall helper on the server.

### CODE REVIEW

File: lua/starfall/libs_sh/pvp.lua [hunk 1]
```diff
@@ -56,64 +64,89 @@ function pvp_library.getBuilders()
 end
 
 -- isInPvp
+--- Returns true if the player is in Pvp mode, false otherwise.
+-- @param ply Player
+-- @return True or False
```
Comment (plally) [inline lua/starfall/libs_sh/pvp.lua:69]: maybe something like `Boolean, is the player is in pvp`?

same for the build and other pvp function

File: lua/starfall/libs_sh/pvp.lua [hunk 2]
```diff
@@ -56,64 +64,89 @@ function pvp_library.getBuilders()
...
 function player_methods:getFactionRank()
     checktype( self, ply_meta )
 
     return getPly( self ):GetNWString( "FactionRank", nil )
 end
 
 -- isInFaction
+--- Returns true if the player is in a faction, false otherwise.
+-- @param ply Player
+-- @return True or false
```
Comment (plally) [inline lua/starfall/libs_sh/pvp.lua:141]:
```suggestion
-- @return Boolean, is the player in a faction
```

File: lua/starfall/libs_sh/pvp.lua [hunk 3]
```diff
@@ -56,64 +64,89 @@ function pvp_library.getBuilders()
...
 function pvp_library.isPlayerInFaction( ply )
     checktype( ply, ply_meta )
 
     return inFaction( getPly( ply ) )
 end
 
+--- Returns true if the player is in a faction, false otherwise.
+-- @return True or false
```
Comment (plally) [inline lua/starfall/libs_sh/pvp.lua:149]:
```suggestion
-- @return Boolean, is the player in a faction
```

File: lua/starfall/libs_sh/pvp.lua (hunk 1 above)
Comment (Syvil) [reply lua/starfall/libs_sh/pvp.lua]: 👍 Done

**Review by brandonsturgeon — APPROVED** [review]

**Pull request merged**